### PR TITLE
CHNG Flu Signal

### PR DESCRIPTION
**Pull request by rumackaaron**
### Description
Add outpatient CHNG flu signal. This signal is analogous to the cli and covid signals already produced from CHNG data. The numerator is ICD codes J09*-J11* and we have already been receiving these aggregated counts from CHNG.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- `load_data.py` - Use `load_flu_data()` to create Pandas df with numerator specified as above and denominator as total outpatient visits.
- Other files were changed as necessary to create the signal but no modifications to the smoothing, adjustments, etc. were made.

### Fixes 
- Implements #1374

### Notes
The pseudocount addition creates a weekday problem. Over the summer, numerators are basically zero for several states while the denominators are smaller on weekends. Adding the pseudocount of 0.5 to the numerator means that there is a spike in the ratio on weekends because the signal value on a weekend is 0.5/(weekend total) and on a weekday is 0.5/(weekday total). This problem is present whether or not weekday adjustment is done, since the weekday adjustment only corrects the numerators which will remain at 0 in these states during the summer.

### CODE REVIEW

File: changehc/delphi_changehc/download_ftp_files.py [hunk 1]
```diff
@@ -93,3 +93,30 @@ def download_cli(filedate, out_path, ftp_conn):
     finally:
         if client:
             client.close()
+
+
+def download_flu(filedate, out_path, ftp_conn):
```
Comment (krivard) [inline changehc/delphi_changehc/download_ftp_files.py:98]: What's the difference between download_flu, download_covid, and download_cli? If they're identical I'd rather we not copy-paste.

**Review by krivard — CHANGES_REQUESTED** [review]
Missing tests; also possible unnecessary code duplication

**Comment by rumackaaron** [comment]
Unduplicated code and added tests

File: changehc/delphi_changehc/download_ftp_files.py [hunk 2]
```diff
@@ -42,34 +42,7 @@ def get_files_from_dir(sftp, filedate, out_path):
 
 
 def download_covid(filedate, out_path, ftp_conn):
```
Comment (krivard) [inline changehc/delphi_changehc/download_ftp_files.py:44]:
```suggestion
def download_counts(filedate, out_path, ftp_conn):
```

File: changehc/delphi_changehc/run.py [hunk 3]
```diff
@@ -14,8 +14,8 @@
 from delphi_utils import get_structured_logger
 
 # first party
-from .download_ftp_files import download_covid, download_cli
-from .load_data import load_combined_data, load_cli_data
+from .download_ftp_files import download_covid
```
Comment (krivard) [inline changehc/delphi_changehc/run.py:17]:
```suggestion
from .download_ftp_files import download_counts
```

File: changehc/delphi_changehc/run.py [hunk 4]
```diff
@@ -26,10 +26,13 @@ def retrieve_files(params, filedate, logger):
...
-            download_covid(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
-        if "cli" in params["indicator"]["types"]:
-            download_cli(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
+        #if "covid" in params["indicator"]["types"]:
+        #    download_covid(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
+        #if "cli" in params["indicator"]["types"]:
+        #    download_cli(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
+        #if "flu" in params["indicator"]["types"]:
+        #    download_flu(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
+        download_covid(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
```
Comment (krivard) [inline changehc/delphi_changehc/run.py:35]:
```suggestion
        download_counts(filedate, params["indicator"]["input_cache_dir"], params["indicator"]["ftp_conn"])
```

**Review by krivard — APPROVED** [review]
👍 

I'm changing the function name but only because I'm insufferable

**Pull request merged**